src/main/constraints/84_interapps_proxy_settings.sql

```diff
@@ -3,8 +3,3 @@ SET search_path = public, pg_catalog;
 ALTER TABLE ONLY interactive_apps_proxy_settings
     ADD CONSTRAINT interactive_apps_proxy_settings_id_pkey
     PRIMARY KEY(id);
```

Comment:
As mentioned in the comment above, primary key constraints have been going into `constraints/00_XX_table_name.sql` files. So we should probably rename this file to `00_84_ interactive_apps_proxy_settings.sql`, since it only contains a `PRIMARY KEY` constraint now.